Assets/HoloToolkit/Input/Scripts/InputSources/CustomInputSource.cs

```diff
@@ -56,6 +56,11 @@ public ButtonStates()
         public bool RaiseEventsBasedOnVisibility;
         public InteractionSourceInfo SourceKind;
 
+        //Navigation Gesture Emulation vars
+        Vector3 NavigatorValues = Vector3.zero; //holds the navigation gesture values [-1,1]
```

Comment:
please explicitly state visibility for class variables

Reply:
changes to this file are from previous commit(s) by other developers.
Its not part of the Tooltip commit.

Reply:
ill make the change anyway because its clear what the effect would be.

Reply:
You may wish to rebase against the may18_dev branch to eliminate files that you are not modifying. the branch was updated to "final" 2017.2.1.4 changes yesterday.

Thanks for being willing to change this, though after the rebase you may not need to :)